README.md

```diff
@@ -1,49 +1,71 @@
-# Header Tool for CodeX Editor
+# Header Tool
 
-Header Editor module for text blocks.
+Header text blocks for [CodeX Editor](https://ifmo.su/editor).
 
-## Install via npm
+## Installation
+
+### Install via NPM
+
+Get the package
 
 ```shell
-npm i --save codex.editor.header
+npm i --save-dev codex.editor.header
 ```
 
-#### Connect with Webpack
-
 Include module in your application
 
-```js
-require('codex.editor.header');
+```javascript
+const Header = require('codex.editor.header');
 ```
 
 Include CSS file
 
 ```css
-@import url("~codex.editor.header/lib/bundle.css");
+@import url("~codex.editor.header/dist/bundle.css");
 ```
 
-## Install directly
+### Load to your project's source dir
 
-1. Download folder `lib` from repository
-2. Add `lib/bundle.js` and `lib/bundle.css` files to your page.
+1. Download folder `dist` from repository
+2. Add `dist/bundle.js` and `dist/bundle.css` files to your page.
 
-## CodeX Editor
+### Load from CDN
 
-API oriented, open-source, block-styled Edtior.
+You can get sources from [RawGit](https://rawgit.com) CDN.
 
-https://github.com/codex-team/codex.editor
+To get the link paste paths to JS ans CSS files and get `production` links to the latest version of Tool.
```

Comment:
Get newest bundle path from RawGit — open [link](https://rawgit.com) and paste ...